### ISSUE TITLE
How to use CRM Keyboard Shortcut in extension shortcuts

### ISSUE THREAD

**Issue by esprout** [SanderRonde/CustomRightClickMenu#81]
What are the CRM Keyboard Shortcut options for in the keyboard shortcuts?  I assigned them to shortcuts, but when I click press the shortcut, nothing happens.  I was hoping they'd trigger the first level menu item.

**Comment by SanderRonde** [comment]
Yeah this works in a pretty unclear way. What you can do with them now is that you can create a script that runs in the background and those scripts can register keyboard handlers [see docs](https://sanderronde.github.io/CustomRightClickMenu/documentation/classes/crm.crmapi.instance.html#background). The idea is to make it possible for CRM items to run outside of a webpage context but still get triggered by keyboard shortcuts.

What you probably want to do though is to trigger some code when you press a shortcut on a page right? For that you can use the [`window.addEventListener('keydown', () => { ... });`](https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event).
